src/piadm.sh

```diff
@@ -0,0 +1,741 @@
+#!/bin/bash
+
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Joyent, Inc.
+#
+
+. /lib/sdc/usb-key.sh
+
+err() {
+    echo $1
```

Comment:
err() etc should be going to stderr

Reply:
Will fix.  Also will introduce `eecho` for cases like ones in the next comment.